runtime/lua/json.lua

```diff
@@ -0,0 +1,383 @@
+--
+-- json.lua
```

Comment:
@bfredl Container allocation functions are not reentrant because they alter a linked list in a number of steps. So it is not safe to call callbacks at random times even without GC run (how, BTW? I do not know how uv is going to perform async read at unsafe time without either using SIGALARM (should probably mess up with our timers), or a separate thread, or a separate process), unless you edit C code of the parser to work with lua containers.

Why not use existing jobs in any case, this is going to be safe for sure? Just have a lambda which immediately calls lua.